GoMeet/src/main/webapp/js/create-meeting-step1.js

```diff
@@ -0,0 +1,41 @@
+let guestsAdded = 0; // Number of guest email inputs on the page 
+
+/**
+ * Add an input element to the guest list for user to add email 
+ * @param {*} doc the document object to be manipulated 
+ */
+function addGuest(doc) {
```

Comment:
I think it's fine if you just use the global `document` value

Reply:
I needed to dependency inject the document object in order to write my unit tests